packages/storybook/src/stories/ino-radio-group/ino-radio-group.stories.ts

```diff
@@ -53,25 +51,43 @@ const template = new TemplateGenerator<Components.InoRadioGroup>(
     </ino-radio-group>
   `
 );
+
 export const Playground = template.generatePlaygroundStory();
 
+Playground.decorators = [story => {
+  updateValueOnChange('#radio-grp1');
+  return story();
+}];
+
 Playground.argTypes = {
   value: {
-    control: {
-      type: 'select',
-    },
+    control: { type: 'select' },
     options: ['opt-1', 'opt-2', 'opt-3'],
   },
+  alignment: {
+    control: { type: 'radio' },
+    options: ['horizontal', 'vertical'],
+  },
 };
 
-export const VerticalAlignment = () => html`
-  <ino-radio-group
-    id="radio-grp"
-    value="opt-2"
-    alignment="vertical"
-  >
-    <ino-radio value="opt-1">Opt 1</ino-radio>
-    <ino-radio value="opt-2">Opt 2</ino-radio>
-    <ino-radio value="opt-3">Opt 3</ino-radio>
-  </ino-radio-group>
-`;
\ No newline at end of file
+const verticalTemplate = new TemplateGenerator<Components.InoRadioGroup>(
+  'ino-radio-group',
+  args => html`
+    <ino-radio-group 
+      id="radio-grp2" 
+      value="${args.value}"
+      alignment="vertical"
+    >
+      <ino-radio value="opt-1">Opt 1</ino-radio>
+      <ino-radio value="opt-2">Opt 2</ino-radio>
+      <ino-radio value="opt-3">Opt 3</ino-radio>
+    </ino-radio-group>
+  `
+);
+
+export const VerticalAlignment = verticalTemplate.generatePlaygroundStory();
+
```

Comment:
It is unnecessary to write a new template for this Story, because the template for both are identical. You can use the template in line 40 and create a new Story with this line

`export const VerticalAlignment = template.generateStoryForProp('alignment', 'vertical')`